### PR TITLE
feat(sbb-calendar, sbb-datepicker-toggle): allow choosing initial calendar view

### LINKED ISSUE #2822: [Feature request]: sbb-calendar - select year first
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### It affects the following packages

Components

### Functionality

**Current behaviour:**
When purchasing as a guest or adding co-passengers in the checkout, the user has to specify the birthdate with the sbb-calendar. When opening the calendar, the date of today is selected by default. 

**Improvement request:**
When the sbb-calender is used for a birthday, the user should be able to select the year as a first step. Afterwards the month and finally the day.

### Link to design proposal file

_No response_

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #2822

### CODE REVIEW

File: src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts [hunk 1]
```diff
@@ -181,4 +183,76 @@ describe(`sbb-datepicker-toggle`, () => {
     expect(changeSpy.count).to.be.equal(1);
     expect(blurSpy.count).to.be.equal(1);
   });
+
+  it('handles view property', async () => {
```
Comment (MarioCastigliano) [inline src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts:187]: Is there a reason that made you test the view progression in the `datepicker-toggle` and the simple "opens on year/month view" in the calendar tests? My gut would tell me to do the other way around 😅

**Review by MarioCastigliano — COMMENTED** [review]
I like this a lot 👍, just one doubt about the test placement

File: src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts (hunk 1 above)
Comment (jeripeierSBB) [reply src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts:187]: It was the main use case and during manual testing, I had more problems there to get everything working. But I can think about it again.

Comment (MarioCastigliano) [reply src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts:187]: My idea was just that the whole selection mechanism working is just up to the calendar component, while the "only" thing the datepicker-toggle has to do is forward the property to the inner calendar. But once again that's just my gut feeling

Comment (jeripeierSBB) [reply src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts:187]: I remember now why I did those tests here.
one use case I can only test here: after closing the calendar by picking a date, it should re-open with the year view again. On the calendar, it just stays on the dayview. 
Furthermore, I could test, if the date of the calendar is still present in the calendar (sync).
The last part with changing to the month view is proving that passing down the change of the "view" property during the run time of the component works.

Comment (MarioCastigliano) [reply src/elements/datepicker/datepicker-toggle/datepicker-toggle.spec.ts:187]: Sounds convincing to me 👍

**Review by MarioCastigliano — APPROVED** [review]

**Pull request merged**